etc/Makefile.am

```diff
@@ -40,5 +40,20 @@ pkgconfig_DATA = flux-core.pc \
 	flux-schedutil.pc
 endif
 
+noinst_SCRIPTS = \
+	completions/flux.in
+
 EXTRA_DIST = \
+	completions/get_builtins.sh \
 	gen-cmdhelp.pl
+
+all: 
+	chmod +x $(srcdir)/completions/get_builtins.sh && \
+	$(srcdir)/completions/get_builtins.sh  $(top_srcdir)/src/cmd/builtin/ \
+	>> completions/flux
```

Comment:
In the automake rule above

 * You shouldn't need to add execute permission to the `get_builtins.sh` script every time. Just make sure the script is checked into git with execute permission...

 * The `all:` rule is run every time, and appends to the `completion/flux` file, so if you run `make` more than once the file gets multiple FLUX_BUILTINS appended to the end.

 * It would be tricky to create a Makefile rule for `completions/flux` since it is already generated by `configure`. I wonder if it would be better to generate `completions/flux.pre` or similar from `configure`, then combine it and output of `get_builtins.sh` to create `completions/flux`, so a rule for `completions_flux` can be used instead of `all`.

Hope that makes sense!